core/trino-main/src/main/java/io/trino/execution/SqlTask.java

```diff
@@ -145,7 +151,7 @@ private SqlTask(
                 // Pass a memory context supplier instead of a memory context to the output buffer,
                 // because we haven't created the task context that holds the memory context yet.
                 () -> queryContext.getTaskContextByTaskId(taskId).localMemoryContext(),
-                () -> notifyStatusChanged(),
+                this::notifyStatusChanged,
```

Comment:
Unrelated change. Move to a separate commit.

Reply:
Sure, done.